bluebrain/repo-bluebrain/packages/py-bglibpy/package.py

```diff
@@ -14,6 +14,7 @@ class PyBglibpy(PythonPackage):
 
     version("develop", branch="main")
     version("4.8.2", commit="0d72a870f39da8201bf0f6dd8171984a89372cff")
+    version("4.10.1", commit="50ec3f984103882f603d48b1a1dacc489d5361da")
```

Comment:
Shouldn't it be 4.10? Or 4.10.0?

Reply:
It is 4.10.1 actually due to the combination of old versioneer and bbp gitlab release pipeline.

If I tag it 4.10.0, then the next commit becomes 4.10.0.1 because of versioneer.
If I want to tag and release 4.10, gitlab pipeline is not happy, it is expecting for the 3rd number.